runtime/src/bank/tests.rs

```diff
@@ -5322,7 +5322,7 @@ fn test_nonce_transaction() {
 
     /* Kick nonce hash off the blockhash_queue */
     for _ in 0..MAX_RECENT_BLOCKHASHES + 1 {
-        goto_end_of_slot(Arc::get_mut(&mut bank).unwrap());
+        goto_end_of_slot(bank.clone());
         bank = Arc::new(new_from_parent(bank));
     }
```

Comment:
i know this `for` is sickeningly copy-pasted below.. but i forgo fixing them all, because i'm lazy. hope i'm doing my part of cleaning job here. ;)